### PR TITLE
Prepare experiment analysis

### PR DESCRIPTION
**Pull request by kirilvasilev16**
Add `apply_cross_validation` method to calculate the training accuracy of a model.
Add `experiments` module that contains a method to read csv data, `read_data`, and a method to plot cross-validation and hold-out accuracy, `plot`.

The pull-request is towards `ingest-data` branch, because I branched from it.

### CODE REVIEW

**Review by AndraIonescu — CHANGES_REQUESTED** [review]

File: experiments/run_experiments.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+from sklearn.model_selection import KFold, train_test_split
+import matplotlib.pyplot as plt
+
+
+def read_data(path, y_column_name):
+    data = pd.read_csv(path)
+    return data.drop([y_column_name], axis=1), data[[y_column_name]]
+
+
+def plot(X, y, max_depth=25, n_splits=10):
```
[the remark this answers is not in the document]
Comment (kirilvasilev16) [reply experiments/run_experiments.py:13]: I assume that the zero-th column is the ID column. Should I add a Python documentation that explains this?

File: experiments/run_experiments.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import numpy as np
+import pandas as pd
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.model_selection import KFold, train_test_split
+import matplotlib.pyplot as plt
+
```
[the remark this answers is not in the document]
Comment (kirilvasilev16) [reply experiments/run_experiments.py:6]: Perhaps `plot_experiments`?

File: experiments/run_experiments.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    ax2.plot(table_holdout.T[0], table_holdout.T[2])
+    ax2.plot(table_holdout.T[0], table_holdout.T[3])
+    ax2.plot(table_holdout.T[0], table_holdout.T[4])
+    ax2.legend(columns)
+    ax2.set_title("Hold-out")
+    ax2.set(xlabel="Decision Tree depth", ylabel="Error")
+
+    fig.set_size_inches(10, 4)
+    fig.show()
+
```
[the remark this answers is not in the document]
Comment (kirilvasilev16) [reply experiments/run_experiments.py]: Created one called `show_plot_experiments`, because if the file name starts or ends with `test`, then a python testing platform is run.

File: experiments/run_experiments.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (AndraIonescu) [reply experiments/run_experiments.py]: Better give the ID column as a parameter to the function.

File: experiments/run_experiments.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (AndraIonescu) [reply experiments/run_experiments.py]: `plot_train_test_accuracy` should be more appropriate.

[the remark this answers is not in the document]
Comment (kirilvasilev16) [reply experiments/run_experiments.py]: Renamed!

**Review by AndraIonescu — APPROVED** [review]
